conference/react-js/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["airbnb"]
+}
```

Comment:
there is a good common styleguide to leave an empty line at the end of each js file. this rule is called eol-last in eslint.
you can decide on your own whether you need it.